packages/portal-components/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@gff/portal-components",
+  "version": "1.0.0",
```

Comment:
Do we want to align this version with other packages for ease?
cc. @craigrbarnes

Reply:
Yeah, it can just be the same as the other packages 👍

Reply:
I agree

Reply:
I noticed that 2.15.5 has been merged to develop.